src/components/Map/CanadaMap.js

```diff
@@ -39,13 +40,13 @@ var CanadaMap = React.createClass({
             numcolorbands: 254,
             version: "1.1.1",
             srs: "EPSG:4326",
-            colorscalerange: "-50,11.0",
+            colorscalerange: "220,320",
             logscale: false
         };
     },
     getWMSParams: function() {
         var params = {layers: this.props.dataset + "/" + this.props.variable};
-        $.extend(params, this.props);
+        _.extend(params, _.pick(this.props, 'noWrap', 'format', 'transparent', 'styles', 'time', 'numcolorbands', 'version', 'srs', 'colorscalerange', 'logscale'));
```

Comment:
This will work well. We can further generalize this, so that it takes a list of param names as an argument. Then we can differentiate between the leaflet initialization params (noWrap, transparent, opacity, etc.) and those that are updateable by the component (styles, time, logscale).